pkgs/development/python-modules/tidalapi/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, python-dateutil
+, requests
+, ffmpeg-python
+, keyring
+, pillow
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "tidalapi";
+  version = "0.7.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-LdlTBkCOb7tXiupsNJ5lbk38syKXeADvi2IdGpW/dk8=";
```

```suggestion
    hash = "sha256-LdlTBkCOb7tXiupsNJ5lbk38syKXeADvi2IdGpW/dk8=";
```

We prefer the more recent hash attribute.